### ISSUE TITLE
Comparision with Hackel Diff algo

### ISSUE THREAD

**Issue by kandelvijaya** [tonyarnold/Differ#51]
Thanks for contributing to the swift community. 

As an author of my own implementation of Diff based on Heckel's algorithm I wanted to know if you did try the comparision with implementation based on Heckel's algo?

Im working on this www.github.com/kandelvijaya/fastdiff while there are few others around. The next immediate one is `Differ` (should be).

**Comment by tonyarnold** [comment]
You'd really have to ask @wokalski - he put together the original implementation, and I wouldn't want to speak on his behalf.

**Comment by wokalski** [comment]
@kandelvijaya the different diff algorithms have different outputs and it'd be nice to make a comparison so that people can use one or another. It'd be even nicer to make all of them conform to the same protocol so that you (as a user) could just switch the underlying implementation with a few lines.

**Comment by wokalski** [comment]
yes, @kandelvijaya. I'm not using swift anymore but capturing 90% of cases first always makes sense.
